data/lm/generate_package.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+from __future__ import absolute_import, division, print_function
+
+# Make sure we can import stuff from util/
+# This script needs to be run from the root of the DeepSpeech repository
+import os
+import sys
+sys.path.insert(1, os.path.join(sys.path[0], '..', '..'))
+
+import argparse
+import shutil
+
+from util.text import Alphabet, UTF8Alphabet
+from ds_ctcdecoder import Scorer, Alphabet as NativeAlphabet
+
+
+def create_bundle(alphabet_path, lm_path, vocab_path, package_path, force_utf8):
+    words = set()
+    vocab_looks_char_based = True
+    with open(vocab_path) as fin:
+        for line in fin:
+            for word in line.split():
+                words.add(word.encode('utf-8'))
+                if len(word) > 1:
+                    vocab_looks_char_based = False
+    print("{} unique words read from vocabulary file.".format(len(words)))
+    print(
+        "{} like a character based model.".format(
+            "Looks" if vocab_looks_char_based else "Doesn't look"
+        )
+    )
+
+    if force_utf8 != None:
+        use_utf8 = force_utf8.value
+    else:
+        use_utf8 = vocab_looks_char_based
```

Comment:
nit: maybe worth a `print()` to help further debug from random people ?

Reply:
Done.